src/test/java/ithillel/lesson5/ArrayHillelListTest.java

```diff
@@ -0,0 +1,56 @@
+package ithillel.lesson5;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArrayHillelListTest {
+    private static final String[] INPUT_ARRAY = {"item1", "item2", "item3"};
+
+    @org.junit.jupiter.api.Test
```

Comment:
```suggestion
    @Test
```
В Java і так назви довгі, а якщо ще і пакети тягнути, то зовсім біда :) 
Краще імпорт пакету зробити.